tests/roles/dataplane_adoption/tasks/nova_verify.yaml

```diff
@@ -9,8 +9,8 @@
   when: prelaunch_test_instance|bool
   ansible.builtin.shell: |
     {{ nova_header }}
-    ${BASH_ALIASES[openstack]} server list | grep -qF '| test | ACTIVE |' && ${BASH_ALIASES[openstack]} server stop test || echo PASS
-    ${BASH_ALIASES[openstack]} server list | grep -qF '| test | SHUTOFF |' || echo FAIL
```

Comment:
set -o pipefail

Reply:
There is a note stating we should not use pipefail here

https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/tests/roles/dataplane_adoption/tasks/nova_verify.yaml#L7

I feel that's the reason we are adding the echo FAIL and custom conditions in the until section to verify the commands.